ironic/drivers/modules/drac/inspect.py

```diff
@@ -35,6 +43,47 @@
 
 METRICS = metrics_utils.get_metrics_logger(__name__)
 
+sushy = importutils.try_import('sushy')
+
+if sushy:
+    CPU_ARCH_MAP = {
+        sushy.PROCESSOR_ARCH_x86: 'x86_64',
+        sushy.PROCESSOR_ARCH_IA_64: 'ia64',
+        sushy.PROCESSOR_ARCH_ARM: 'arm',
+        sushy.PROCESSOR_ARCH_MIPS: 'mips',
+        sushy.PROCESSOR_ARCH_OEM: 'oem'
+    }
+
+    BOOT_MODE_MAP = {
+        sushy.BOOT_SOURCE_MODE_UEFI: boot_modes.UEFI,
+        sushy.BOOT_SOURCE_MODE_BIOS: boot_modes.LEGACY_BIOS
+    }
```

Comment:
Can we just use the constants defined in the superclass instead of redefining them here?

Reply:
maybe redfish_inspect.CPU_ARCH_MAP?

Reply:
Done